app/src/main/java/com/example/placefinder/RetrofitClientInstance.kt

```diff
@@ -4,7 +4,7 @@ import retrofit2.converter.gson.GsonConverterFactory
 
 object RetrofitClientInstance {
     private var retrofit: Retrofit? = null
-    private val BASE_URL = "foo"
+    private val BASE_URL = "https://developer.nps.gov/"
```

Comment:
Added the base URL of your JSON service to your RetrofitClientInstance.  I followed this video created by the Professor to add the URLs: https://www.youtube.com/watch?v=u9fRy2N9GzE